lib/specs.js

```diff
@@ -1022,45 +887,18 @@ var validateSemantically = function (spec, rlOrSO, apiDeclarations, callback) {
   var cbWrapper = function (err, results) {
     callback(err, helpers.formatResults(results));
   };
-  if (spec.version === '1.2') {
```

Comment:
I can't comment on non modified lines but seems that there are more conditional logic in this file related to version 1.2 that can be cleaned.

Reply:
The main point of this is to remediate the security issues.  There is still a fair bit of residual '1.2' laying around.  The reason is that I didn't want to make too many structural changes.  It's not to produce a perfect/clean version with tabs, const, etc.  The '1.2' changes I made were either necessary (to get rid of the conversion module they were using), or in some cases, increased complexity in areas of code that I was looking at because there were issues around it.